src/utils/SpokeUtils.ts

```diff
@@ -393,3 +395,7 @@ export async function findFillBlock(
 export function isZeroAddress(address: string): boolean {
   return address === ZERO_ADDRESS || address === ZERO_BYTES;
 }
+
+export function getMessageHash(message: string): string {
```

Comment:
nb should add a couple of tests for this.

Reply:
Added: https://github.com/across-protocol/sdk/pull/851/commits/d051eaf006b67d24a1571dfc243c1dc382da0dee